src/react/src/components/DownloadFacilitiesButton.jsx

```diff
@@ -56,19 +57,18 @@ function DownloadFacilitiesButton({
     classes,
     theme,
 }) {
-    const [requestedDownload, setRequestedDownload] = useState(false);
     const [anchorEl, setAnchorEl] = React.useState(null);
 
     const actionContrastText = theme.palette.getContrastText(
         theme.palette.action.main,
     );
 
     useEffect(() => {
-        if (requestedDownload && logDownloadError) {
-            toast('A problem prevented downloading the facilities');
-            setRequestedDownload(false);
+        if (logDownloadError) {
```

Comment:
If it is an array, it should be `if (Array.isArray(logDownloadError) && logDownloadError.length > 0)`